### PR TITLE
Fix popup window doesn't not displaying text, if text contains special UTF-8 symbols

### LINKED ISSUE #400: Sometimes, "Translate selected text" pop up a textbox with no text, depending on how I selected it.
**Bug description**
Mostly on Reddit comments, if I select an entire paragraph with two clicks, the translation appears empty when pressing the "Translate selected text" hotkey. It works if I manually select the text.

**Steps to reproduce**
1. Select an entire paragraph of some Reddit comment by clicking twice.
2. Press the "Translate selected text" hotkey.

**Expected behavior**
Some kind of anticipation regarding what may change when using each way to select the text, clicking twice, or manually selecting it.

**Screenshots**
Vídeo -> https://imgur.com/a/ffdfe74

**Environment**

- OS: Arch Linux
- Desktop environment:  KDE
- Crow Translate version: 2.9.1

### LINKED ISSUE #416: Translation does not work if there is anything other than text in the text
**Bug description**
If the text contains extraneous entities, then the text is not translated at all

**Steps to reproduce**
- [ ] try to translate this text by highlighting the checkbox including

**Expected behavior**
Get a normal translation

**Screenshots**
image

**Environment**
- OS: Arch Linux
- Desktop environment: awesome
- Crow Translate version: 2.9.2

**Additional context**
This problem also occurs if you try to translate text with emoji in discord

### PR DESCRIPTION
**Pull request by VolkMilit**
Fixes #416 and fixes #400

### CODE REVIEW

File: src/translationedit.cpp [hunk 1]
```diff
@@ -42,28 +42,28 @@ bool TranslationEdit::parseTranslationData(QOnlineTranslator *translator)
     m_lang = translator->translationLanguage();
 
     // Translation
-    setHtml(m_translation.toHtmlEscaped().replace(QStringLiteral("\n"), QStringLiteral("<br>")));
+    QString resultTranslation = m_translation.toHtmlEscaped().replace(QStringLiteral("\n"), QStringLiteral("<br>"));
```
Comment (Shatur) [inline src/translationedit.cpp:45]:
```suggestion
    QString formattedTranslation = m_translation.toHtmlEscaped().replace(QStringLiteral("\n"), QStringLiteral("<br>"));
```

Comment (VolkMilit) [reply src/translationedit.cpp]: Done.

File: src/translationedit.cpp [hunk 2]
```diff
@@ -82,36 +82,38 @@ bool TranslationEdit::parseTranslationData(QOnlineTranslator *translator)
...
+                formattedTranslation += QStringLiteral("<font color=\"grey\"><i>%1</i></font>").arg(example);
+                formattedTranslation += "<br>";
             }
             indent.setTextIndent(0);
             textCursor().setBlockFormat(indent);
         }
     }
 
+    setHtml(formattedTranslation);
+
```
Comment (Shatur) [inline src/translationedit.cpp:114]: Nitpick: could you please remove this line to alight setHtml with `moveCursor`?
```suggestion
```

Comment (VolkMilit) [reply src/translationedit.cpp]: Done.

**Comment by rizzini** [comment]
It's happening again since v2.9.11. I just compiled the v2.9.10, and it's working. It must be something with https://github.com/crow-translate/crow-translate/commit/7e64d2c382fbbc94ef04f6b83eb291fdf63ee289, right?

**Comment by VolkMilit** [comment]
@rizzini, yes, I'm fix this in 49dec0e397ea8947afbae02c8962d75dd68c7ddc

**Pull request merged**